### ISSUE TITLE
Add Twitter follow buttons and Facebook like buttons for pages

### ISSUE THREAD

**Issue by danielpetty** [denverpost/dptv#106]
Ideally, right underneath the Sports Show logo on all pages, between the main logo and the main player.

**Comment by freejoe76** [comment]
Did you get @utahras' blessing on this? That's a prime piece of real estate.

**Comment by danielpetty** [comment]
I can't tag @UtahRas in anything. So I assume he's OK with it.

**Comment by freejoe76** [comment]
That's not the same as getting his blessing. On hold until blessed.

**Comment by freejoe76** [comment]
Will close this issue at end of the week unless @utahras gives the okay on adding follow buttons under Sports Show logo.

**Comment by danielpetty** [comment]
This was Ok'd by Rasmussen, Risch, Petty during Jan. 29 meeting.

**Comment by freejoe76** [comment]
Super, can you get me the markup you want on the page?

**Comment by freejoe76** [comment]
Sent screenshot of follow button look to stakeholders.

**Comment by freejoe76** [comment]
Waiting on Facebook URLs for DPTV and Sports Show

**Comment by freejoe76** [comment]
Will generate FB follow button ( https://developers.facebook.com/docs/plugins/like-button ) when those come in.

**Comment by freejoe76** [comment]
The twitter follow button is live on DPTV.

**Comment by freejoe76** [comment]
WONTFIX
